### PR TITLE
:mortar_board: Topic5b Example Cleanup

### LINKED ISSUE #5: Update example in Class/Topic 5b
The example's actual implementation can be improved upon. It's pretty janky at the moment.

### PR DESCRIPTION
**Pull request by jameshughes89**
### Related Issues or PRs
Closes #5 

### What
Just clean up with white space for student readability. 
Fix a typo. 
Remove 'screw-em' comment. 
Remove periods from points because I have been excluding them everywhere else for some reason :telescope: 

### Why
Just a clean up. For some reason I thought the example was worse than it is.

### CODE REVIEW

File: source/topic5b.rst [hunk 1]
```diff
@@ -197,7 +210,7 @@ Now for the tough one... calculate the total cost. What do we know?
 
 **Hmm, defo was tricker, but still not too bad at all!**
 
-Now just do the IO part, which is easy. 
+Now just do the IO part, which is easy
```
Comment (twentylemon) [inline source/topic5b.rst:213]: easy -> simple

**Review by twentylemon — APPROVED** [review]

File: source/topic5b.rst (hunk 1 above)
Comment (jdelamer) [reply source/topic5b.rst:213]: I agree, simple is better.

**Comment by jameshughes89** [comment]
Try now

**Comment by twentylemon** [comment]
> Try now

I mean. I prefer simple. But still :approved:

**Pull request merged**